metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -146,9 +146,9 @@ func getEnvPort() string {
 // GetConfig returns config for elasticsearch module
 func getConfig(metricset string) map[string]interface{} {
 	return map[string]interface{}{
-		"module":                     elasticsearch.ModuleName,
-		"metricsets":                 []string{metricset},
-		"hosts":                      []string{getEnvHost() + ":" + getEnvPort()},
+		"module":     elasticsearch.ModuleName,
```

Comment:
That diff should not be here. What golang version do you have locally? Make sure you are on 1.11.(5)

Reply:
I have fmt 5.3.0 on the host, and go was 1.10.X
I installed go 1.11.5, run fmt and will push again